.circleci/config.yml

```diff
@@ -984,6 +984,33 @@ jobs:
             # Upload to GitHub
             ./ghr -replace ${CIRCLE_TAG} glean-core/python/dist
 
+  ###########################################################################
+  # C#
+
+  CSharp tests:
+    docker:
+      - image: mcr.microsoft.com/dotnet/core/sdk:3.1-buster
```

Comment:
Just to understand: this is an actual Linux image, just with all the dotnet tools in there?

Reply:
Yes! This is a Debian image to be precise. The list of images is [available here](https://hub.docker.com/_/microsoft-dotnet-core-sdk/).